addons/website/tests/test_ui.py

```diff
@@ -155,3 +155,56 @@ def test_06_public_user_editor(self):
 
     def test_07_carousel_snippet_content_removal(self):
         self.start_tour("/", "carousel_content_removal", login='admin')
+
+    def test_08_focus_blur_unit_test(self):
```

Comment:
```suggestion
    def test_08_editor_focus_blur_unit_test(self):
```
Minor: Only if you were to change anything else related to the `content`key, otherwise keep it as such